lib/grammar.pegjs

```diff
@@ -787,6 +787,13 @@ short_date_value = ('makeDate' / 'new' __ 'Date') _ '(' _ year:literal_number? _
     d.setMilliseconds(0);
     return new Ast.Value.Date(d);
 }
+long_weekday_value = 'new' __ 'Date' _ '(' _ day:quoted_weekday _ ',' _ hours:literal_number _ ',' _ minutes:literal_number _ ',' _ seconds:literal_number _ ')' {
```

Comment:
Do you really want the weekday to be quoted? Why?

Also, don't do this: use `ident` or `literal_string` and throw a type error if the day is incorrect. You need to anyway because the NN syntax doesn't check that the day is valid, and that way you don't get an obscure syntax error at the beginning of the whole expression.